app/repositories/user_profile_repository.py

```
@@ -0,0 +1,317 @@
+        await session.refresh(profile)
+
+        logger.debug(f"📝 사용자 {user_id} 프로필 업데이트 완료")
+        return profile
+
+    @staticmethod
+    async def get_profiles_with_vectors(
+        session: AsyncSession,
+        exclude_user_id: Optional[int] = None
+    ) -> List[UserProfile]:
+        """벡터가 있는 프로필들을 조회합니다."""
+        conditions = [UserProfile.profile_vector != None]  # pylint: disable=singleton-comparison
```

Ruff (0.11.9)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)